applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_compressible_potential_element.cpp

```diff
@@ -72,5 +72,108 @@ namespace Kratos {
 			KRATOS_CHECK_NEAR(RHS(1), 0.0, 1e-7);
 			KRATOS_CHECK_NEAR(RHS(2), -0.5, 1e-7);
 		}
+
+		/** Checks the CompressiblePotentialFlowElement element.
+	     * Checks the EquationIdVector.
+	     */
+	    KRATOS_TEST_CASE_IN_SUITE(CompressiblePotentialFlowElement_EquationIdVector, CompressiblePotentialApplicationFastSuite)
+		{
+
+			ModelPart modelPart("Main");
+
+			// Variables addition
+			modelPart.AddNodalSolutionStepVariable(POSITIVE_FACE_PRESSURE);		
+
+			// Set the element properties
+			Properties::Pointer pElemProp = modelPart.pGetProperties(0);
+
+			// Geometry creation
```

Comment:
You might want to put the parts that are not directly part of the test (here I mean the construction of the model part with an element) into a separate function.

Reply:
Done. Ill push it when I finish including all of the changes.